concent_api/core/migrations/0003_add_and_update_deadline_and_size.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-06-19 09:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+from common.helpers import deserialize_message
+from common.helpers import parse_timestamp_to_utc_datetime
+
+
+def populate_subtask_size_field(apps, _schema_editor):
+    Subtask = apps.get_model('core', 'Subtask')
+    StoredMessage = apps.get_model('core', 'StoredMessage')
+    for subtask in Subtask.objects.filter(result_package_size__isnull=True):
+        if StoredMessage.objects.filter(
```

Comment:
Please fix styling.

Reply:
You can achieve this in single query, using `first()`, then checking if returned value is not None.

Reply:
Done.